app/main.py

```diff
@@ -242,6 +250,56 @@ def _retrieve_permanent_location(holdings_id: str, okapi_headers: dict) -> tuple
     return permanent_location_id, location_name, instance_id
 
 
+def _set_callno_type(holdings: dict, xml: etree._XSLTResultTree, okapi_headers: dict):
+    """
+    Updates Call Number Type desc property and value based on 
+    """
+    call_number_uuid = holdings['callNumberTypeId']
+    call_number_type_result = requests.get(
+        f"{os.getenv('OKAPI_URL')}/call-number-types/{call_number_uuid}",
+        headers=okapi_headers
+    )
+    call_number_type_result.raise_for_status()
+    name = call_number_type_result.json()["name"]
+    code = None
+    # Matches Ex Libris codes and names at https://developers.exlibrisgroup.com/alma/apis/docs/xsd/rest_item.xsd/?tags=GET#holding_data
```

Comment:
@ahafele - the following names and codes are from the Ex Libris link, does this align with the what the users are expecting?